src/dftbp/dftb/sccinit.F90

```diff
@@ -555,8 +556,8 @@ subroutine initQFromFile(qq, fileName, tReadAscii, orb, qBlock, qiBlock, density
           else
             read(file%unit, iostat=iErr) coeffsAndShifts
           end if
-          call checkSupercellFoldingMatrix(coeffsAndShifts, errStatus,&
-              & supercellFoldingDiagOut=supercellFoldingDiag)
+          call checkSupercellFoldingMatrix(coeffsAndShifts, errStatus)
```

Comment:
Don't we have to handle the error here in some way?

Reply:
Yes (wasn't handled previous either)

Reply:
I know :see_no_evil:, thanks for fixing it.